.github/workflows/publish-packages-automatically.yaml

```diff
@@ -55,7 +55,6 @@ jobs:
         uses: ./.github/workflows/publish-common-package.yaml
         with:
           npm_access_token: ${{ secrets.NPM_ACCESS_TOKEN }}
-          version_changed: ${{ needs.prepare.outputs.common_changed }}
```

Comment:
@cristidas automatic shouldn't run on every dev commit, as it needs to check changes in the package.json version to make sense

Reply:
@piotrwitek it still checks for changes [here](https://github.com/OasisDEX/summerfi-monorepo/blob/84a93c70846169b565e3306ea0a10a1f8d8c64f6/.github/workflows/publish-packages-automatically.yaml#L46) and [here](https://github.com/OasisDEX/summerfi-monorepo/blob/84a93c70846169b565e3306ea0a10a1f8d8c64f6/.github/workflows/publish-packages-automatically.yaml#L63). I removed the `version_changed` part because it was duplicating the check